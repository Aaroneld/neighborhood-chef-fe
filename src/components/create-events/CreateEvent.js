import React from 'react';
import FormContainer from './form-container/FormContainer';

const CreateEvent = () => {
    return (
        <div className="create-event-main">
            <FormContainer />
        </div>
    );
};

export default CreateEvent;
