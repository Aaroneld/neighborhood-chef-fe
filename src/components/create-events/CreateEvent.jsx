import React from 'react';
import FormContainer from './form-container/FormContainer';

const CreateEvent = () => {
  return (
    <div style={{ display: 'flex', justifyContent: 'center' }}>
      <FormContainer />
    </div>
  );
};

export default CreateEvent;
