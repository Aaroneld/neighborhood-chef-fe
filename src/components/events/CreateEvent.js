import React from "react";
import Sidebar from "../dashboard/Sidebar";
import Header from "../dashboard/Header";

const CreateEvent = () => {
  return (
    <div>
      <Header />
      <h1>CreateEvent Component</h1>
      <Sidebar active="createEvent" />
    </div>
  );
};

export default CreateEvent;
