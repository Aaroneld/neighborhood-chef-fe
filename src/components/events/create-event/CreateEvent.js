import React from "react";
import FormContainer from "./FormContainer";

const CreateEvent = () => {
  return (
    <div className="component-main">
      <FormContainer />
    </div>
  );
};

export default CreateEvent;
