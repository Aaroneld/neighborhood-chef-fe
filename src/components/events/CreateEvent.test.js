import React from 'react';
import CreateEvent from './CreateEvent.js';
import renderer from 'react-test-renderer';

it('CreateEvent component renders', () => {
    const CreateEventComponent = renderer.create(<CreateEvent />).toJSON();
    expect(CreateEventComponent).toMatchSnapshot();
});
