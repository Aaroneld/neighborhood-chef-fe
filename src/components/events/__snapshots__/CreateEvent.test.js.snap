// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateEvent component renders 1`] = `
<h1>
  CreateEvent Component
</h1>
`;
